python/pyspark/ml/classification.py

```diff
@@ -1032,6 +1032,16 @@ class NaiveBayes(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, H
     >>> result = model3.transform(test0).head()
     >>> result.prediction
     0.0
+    >>> dfWithWeight = spark.createDataFrame([
```

Comment:
Sorry for the confusion, I was thinking we could just add weights to the existing example. I don't think an entirely separate example is necessary. What do you think?